sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -444,3 +448,123 @@ def display_data(self):
 
   def writer(self):
     raise NotImplementedError
+
+
+class MultipleReadFromPubSub(PTransform):
+  """A ``PTransform`` that expands ``ReadFromPubSub`` to read from multiple
+  subscriptions and/or topics."""
+  def __init__(
+      self,
+      source_list,  # type: List[str]
+      with_context=False,  # type: bool
+      id_label=None,  # type: Optional[Union[List[str], str]]
+      with_attributes=False,  # type: bool
+      timestamp_attribute=None,  # type: Optional[Union[List[str], str]]
+      **kwargs
```

Comment:
It also seems like there is no certain consumer for these additional `kwargs`. Why do we want them here?